packages/headless/src/controllers/smart-snippet/headless-smart-snippet.ts

```diff
@@ -1,192 +1,14 @@
-import {QuestionAnswerDocumentIdentifier} from '../../api/search/search/question-answering';
-import {Result} from '../../api/search/search/result';
-import {search, questionAnswering} from '../../app/reducers';
 import {SearchEngine} from '../../app/search-engine/search-engine';
+import {smartSnippetAnalyticsClient} from '../../features/question-answering/question-answering-analytics-actions';
 import {
-  closeFeedbackModal,
-  collapseSmartSnippet,
-  dislikeSmartSnippet,
-  expandSmartSnippet,
-  likeSmartSnippet,
-  openFeedbackModal,
-} from '../../features/question-answering/question-answering-actions';
-import {
-  logCollapseSmartSnippet,
-  logDislikeSmartSnippet,
-  logExpandSmartSnippet,
-  logLikeSmartSnippet,
-  logOpenSmartSnippetSource,
-  logOpenSmartSnippetFeedbackModal,
-  logCloseSmartSnippetFeedbackModal,
-  logSmartSnippetFeedback,
-  logSmartSnippetDetailedFeedback,
-  SmartSnippetFeedback,
-} from '../../features/question-answering/question-answering-analytics-actions';
-import {answerSourceSelector} from '../../features/question-answering/question-answering-selectors';
-import {pushRecentResult} from '../../features/recent-results/recent-results-actions';
-import {QuestionAnsweringSection} from '../../state/state-sections';
-import {loadReducerError} from '../../utils/errors';
-import {buildController, Controller} from '../controller/headless-controller';
-import {buildInteractiveResultCore} from '../core/interactive-result/headless-core-interactive-result';
-import {
-  buildSmartSnippetInteractiveInlineLinks,
-  InlineLink,
-} from './headless-smart-snippet-interactive-inline-links';
+  SmartSnippet,
+  SmartSnippetProps,
+  buildCoreSmartSnippet,
+} from '../core/smart-snippet/headless-core-smart-snippet';
+import {buildSmartSnippetInteractiveInlineLinks} from './headless-smart-snippet-interactive-inline-links';
 
 export type {QuestionAnswerDocumentIdentifier} from '../../api/search/search/question-answering';
 
-export interface SmartSnippetOptions {
```

Comment:
ok I understand all this is what was moved into the headless-core-smart-snippet?